### DISCUSSION TITLE
Request for feedback: Frieda,  a code generator for the fetch driver

### DISCUSSION THREAD

**Discussion by cdcarson** [planetscale/discussion#511] [General]
I've open-sourced some code I've been using for a while: [Frieda](https://github.com/cdcarson/frieda). It's code generator for the [PlanetScale's fetch driver](https://github.com/planetscale/database-js). It's not another ORM, it's not a query builder. It just takes your database schema and turns it into well-typed code that your javascript/typescript application can easily consume.

All sorts of feedback is welcome. I'd love for folks to try it out, but you can also agree with my chihuahua that it has been a complete [waste of time](https://github.com/cdcarson/frieda#whos-frieda).

**Comment by mscoutermarsh** [comment]
Thanks for sharing, say Hi to Frieda for us. 🐕
